src/DynamoCore/Configuration/PathManager.cs

```diff
@@ -686,6 +686,16 @@ private IEnumerable<string> LibrarySearchPaths(string library)
             yield return Path.GetFullPath(library);
         }
 
+        internal static string GetAppDataFolder()
```

Comment:
What is the difference of this function v.s. e.g. `GetCommonDataFolder`?

Reply:
Got it, using the Dynamo View Model reference instead